Tests/L0/SonarQubePostTest/SummaryReport.ps1

```diff
@@ -22,59 +22,60 @@ function VerifyMessage
+    Assert-AreEqual $expectedThreshold $actualMessage.threshold "Invalid message threshold"    
     Assert-AreEqual $expectedComparator $actualMessage.comparator "Invalid message comparator"    
 }
 
-
-#### Test 1 - Legacy report - uploading fails with a warning if the file is not found
+# #### Test 1 - Legacy report - uploading fails with a warning if the file is not found
     
-# Arrange
-Register-Mock GetSonarQubeOutDirectory {([IO.Path]::GetRandomFileName())}
-Register-Mock Write-Warning 
-Register-Mock IsPrBuild {$true}  
+# # Arrange
+# Register-Mock GetSonarQubeOutDirectory {([IO.Path]::GetRandomFileName())}
+# Register-Mock Write-Warning 
+# Register-Mock IsPrBuild {$true}  
 
-# Act
-CreateAndUploadReport
+# # Act
+# CreateAndUploadReport
 
-# Assert
-Assert-WasCalled Write-Warning 
+# # Assert
+# Assert-WasCalled Write-Warning 
 
-# Cleanup
-Unregister-Mock GetSonarQubeOutDirectory
-Unregister-Mock Write-Warning
-Unregister-Mock IsPrBuild   
+# # Cleanup
+# Unregister-Mock GetSonarQubeOutDirectory
+# Unregister-Mock Write-Warning
+# Unregister-Mock IsPrBuild   
 
 
-#### Test 2 - Legacy report upload: the command responsible for uploading the summary report is called  
+# #### Test 2 - Legacy report upload: the command responsible for uploading the summary report is called  
 
-# Arrange
+# # Arrange
   
-$tempDir = CreateRandomDir
-$dummyReport = [IO.Path]::Combine($tempDir, "summary.md") 
-$file = [IO.File]::Create($dummyReport)   
+# $tempDir = CreateRandomDir
+# $dummyReport = [IO.Path]::Combine($tempDir, "summary.md") 
+# $file = [IO.File]::Create($dummyReport)   
 
-Register-Mock GetSonarQubeOutDirectory {$tempDir}
-Register-Mock Write-Host
-Register-Mock IsPrBuild {$false}  
-# the legacy report is printed if the user deselects the full report option
-Register-Mock GetTaskContextVariable {$false} -- "MSBuild.SonarQube.IncludeReport" 
+# Register-Mock GetSonarQubeOutDirectory {$tempDir}
+# Register-Mock Write-Host
+# Register-Mock IsPrBuild {$false}  
+# # the legacy report is printed if the user deselects the full report option
+# Register-Mock GetTaskContextVariable {$false} -- "MSBuild.SonarQube.IncludeReport" 
 
-# Act
-CreateAndUploadReport
+# # Act
+# CreateAndUploadReport
 
-# Assert
-Assert-WasCalled Write-Host -ArgumentsEvaluator { $args[0].StartsWith("##vso[task.addattachment type=Distributedtask.Core.Summary;name=SonarQube Analysis Report;") }
+# # Assert
+# Assert-WasCalled Write-Host -ArgumentsEvaluator { $args[0].StartsWith("##vso[task.addattachment type=Distributedtask.Core.Summary;name=SonarQube Analysis Report;") }
 
-# Cleanup
-$file.Dispose()
-[IO.Directory]::Delete($tempDir, $true)    
-Unregister-Mock GetSonarQubeOutDirectory
-Unregister-Mock Write-Host 
-Unregister-Mock IsPrBuild   
+# # Cleanup
+# $file.Dispose()
+# [IO.Directory]::Delete($tempDir, $true)    
+# Unregister-Mock GetSonarQubeOutDirectory
+# Unregister-Mock Write-Host 
+# Unregister-Mock IsPrBuild   
```

Comment:
It may be helpful to add comments explaining why these are now ignored - or remove them completely if there is no longer any value to them.